my-react-app/src/redux/login/actions.js

```diff
@@ -13,13 +13,11 @@ const actionCreators = {
     if (response.ok) {
       dispatch({
         type: actions.GET_TOKEN_SUCCESS,
-        payload: response.data
+        payload: { token: response.data, email: response.data }
```

Comment:
It should be `token: response.data.token`, as data is an object that has a `token` property